proto/resources/internal/stats/key_figure_card.proto

```diff
@@ -14,21 +14,21 @@ message KeyFigureCard {
 message KeyFigure {
 
   enum Rank {
-    UNSPECIFIED = 0;
+    UNSPECIFIED_RANK = 0;
     PRIMARY = 1;
     SECONDARY = 2;
     TERTIARY = 3;
   }
 
   enum Trend {
-    NONE = 0;
+    UNSPECIFIED_TREND = 0;
     STABLE = 1;
     INCREASING = 2;
     DECREASING = 3;
   }
 
   enum TrendSemantic {
-    NONE = 0;
+    UNSPECIFIED_TREND_SEMANTIC = 0;
```

Comment:
No need to suffix the value here, as it has context from the enum name.
``` suggestion
    UNSPECIFIED = 0;
``

Reply:
They actually do have to be unique. That's super annoying, otherwise it doesn't compile:

Error message:
> Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, "NONE" must be unique within "app.coronawarn.server.common.protocols.internal.stats", not just within "TrendSemantic".


see extensive commit message https://github.com/corona-warn-app/cwa-protocol-buffers/commit/24233fcb351668f1cdbecf972ada616acd22bb7d

Reply:
🙄  oh well, this is so 2000…
Anyways, thanks for the hint :)